src/foam/mlang/mlangJava.js

```diff
@@ -350,6 +350,21 @@ foam.CLASS({
 });
 
 
+foam.CLASS({
+  refines: 'foam.mlang.predicate.Has',
+
+  methods: [
+    {
+      name: 'f',
+      javaCode: 'Object value = getArg1().f(obj); '+
```

Comment:
@kgrgreer we've started using backtick strings for documentation. Should we allow/encourage the same for generated code strings?